public/styles/chat.css

```diff
@@ -90,6 +94,7 @@ header {
 	.message {
 		display: flex;
 		align-items: baseline;
+		word-break: break-word;
```

Comment:
This results in inconsistencies when it comes to cross-browser support. In Firefox, the username will be above the message, whereas in Chrome the username is to the left of the message. I would suggest the following as a fix:

```css
.content {
  white-space: nowrap;
}
```

```css
.text {
  word-break: break-word;
  white-space: normal;
}